metadata/tests/unit/proxy/test_neo4j_proxy.py

```diff
@@ -1456,5 +1460,65 @@ def test_get_resource_generation_code_not_found(self) -> None:
                               resource_type=ResourceType.Feature)
 
 
+class TestNeo4jProxyHelpers:
```

Comment:
The above class feels more like an integration test. Created this one for testing helpers.